src/main/java/com/solvd/movie/web/dto/mapper/SearchCriteriaMapper.java

```diff
@@ -5,8 +5,10 @@
 import org.mapstruct.Mapper;
 
 @Mapper(componentModel = "spring")
-public interface SearchCriteriaMapper
-        extends com.solvd.movie.web.dto.mapper.Mapper<SearchCriteria,
-        SearchCriteriaDto> {
+public interface SearchCriteriaMapper {
+
+    SearchCriteria toEntity(SearchCriteriaDto dto);
```

Comment:
@hizmailovich what is the diff between `SearchCriteria` and `SearchCriteriaDto`?

Reply:
There are no diff between these classes, but I think that if I am using DTO pattern, It will be better to implement dto class for criteria too